### PR TITLE
add options for tmux and screen

### PR DESCRIPTION
**Pull request by Upow1234**
Addresses #32.  Add options for using tmux and screen. Can set default variables for window orientation and window size in .vimrc and/or set them on the fly using :call SClangStart([options]). Updated readme with detailed instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ Change it if you have installed the plugin in other location.
 Defaults to `"~/.vim/bundle/scvim/bin/sc_dispatcher"`.
 Change it if you have installed the plugin in other location.
 * `g:scFlash`: Highlighting of evaluated code
+* `g:sclangWindowOrientation`: sets default window orientation when using
+a supported terminal multiplexer. See Multiplexer Options for more info
+* `g:sclangWindowSize`: sets default window size when using a supported
```
Comment (gusano) [inline README.md]: @Upow1234 "..sets default window size **(% of screen)** ..."

File: README.md [hunk 2]
```diff
@@ -119,3 +123,54 @@ in normal/insert mode:
...
 * `F12` is a hard stop
+
+Terminal Multiplexer Options
+-----
+
+Supported Terminal Multiplexers are 'tmux' and 'screen'. To use with scvim,
+open the multiplexer before opening vim.  
+For example:
+
+`user@pc-555: tmux`
```
Comment (gusano) [inline README.md]: @Upow1234 I would recommend using a session (good habit): `tmux new -s sc`

File: README.md [hunk 3]
```diff
@@ -119,3 +123,54 @@ in normal/insert mode:
...
+open the multiplexer before opening vim.  
+For example:
+
+`user@pc-555: tmux`  
+`user@pc-555: vim mySCfile.scd`  
+
+Default settings for window orienation and window size can be set in
+your .vimrc file. 
+
+Window orientation options are "h" for horizonatal and "v" for veritcal.
```
Comment (gusano) [inline README.md]: @Upow1234 2 typos: `horizonatal` and `veritcal` :)

**Comment by Upow1234** [comment]
Hello again,

Sorry for the delay...I commited the changes you requested to the bugfix/issue_32 branch. I can see them reflected in the commentary above. Do I need to do another pull request? I am new to this and am trying to do the right thing. 

I am happy to contribute and look forward to doing more in the future.

**Comment by gusano** [comment]
@Upow1234 sorry for the very late reply, but this doesn't seem to work for me.
Testing with `tmux` and no defaults in my `.vimrc` I get the following:

deepinscreenshot_select-area_20171205104717

As you can see, my main split is on the top and super tiny (text is not even visible) while the post split takes up all space.
I didn't try `screen` as I don't use it.

**Comment by gusano** [comment]
@Upow1234 another update:

- tmux unit for split size is always `%`
- screen defaults to lines number but **can be told to resize in `%` as well** 
- the size given is the size of the post window

So what I'd propose for this PR is to:

- [ ] use `%` units for split sizes both for `tmux` and `screen` (less confusing)
- [ ] fix `screen` implementation so both `tmux` and `screen` use `h` for horizontal split and `v` for vertical split
- [ ] both `screen` and `tmux` should use the same defaults (my vote would be `h` `50` (%))
- [ ] update the docs so it's slightly less confusing

**Comment by gusano** [comment]
Closing in favor of #38

**Pull request closed without merging**